core/src/com/unciv/models/gamebasics/tile/Terrain.kt

```diff
@@ -33,6 +33,9 @@ class Terrain : NamedStats(), ICivilopedia {
                 sb.appendln("{Defence bonus}: ".tr()+(defenceBonus*100).toInt()+"%")
             }
 
+            if(rough)
+                sb.appendln("{Rough Terrain}: ".tr() + if(rough) "yes".tr() else "no".tr())
```

Comment:
Why is there an if on yes/no if you only have this line when yes to begin with?

Reply:
I'm not sure what you are trying to say :)

Do you see a potential problem here? How would you like to change this line?